tests/regression_tests/dagmc/external/main.cpp

```diff
@@ -0,0 +1,94 @@
+#include "openmc/capi.h"
+#include "openmc/cross_sections.h"
+#include "openmc/dagmc.h"
+#include "openmc/error.h"
+#include "openmc/geometry.h"
+#include "openmc/geometry_aux.h"
+#include "openmc/material.h"
+#include "openmc/nuclide.h"
+#include <iostream>
+
+int main(int argc, char* argv[])
+{
+  using namespace openmc;
+  int openmc_err;
+
+  // Initialise OpenMC
+  openmc_err = openmc_init(argc, argv, nullptr);
+  if (openmc_err == -1) {
+    // This happens for the -h and -v flags
```

Comment:
Can you elaborate on this comment? Is this something we should look into?

Reply:
This comment is copy-pasted verbatim from OpenMC's main.cpp L24 ... You tell me.

Reply:
Took a look. Going to update this to be more clear in another PR.